src/windows.hpp

```diff
@@ -51,6 +51,7 @@
 #include <winsock2.h>
 #include <windows.h>
 #include <mswsock.h>
+#include <iphlpapi.h>
```

Comment:
Please don't re-add iphlpapi, this was recently reverted because it breaks compatibility with supported build targets on Windows.